Modules/System Tests/Translation/src/TranslationTestTableTwo.Table.al

```diff
@@ -0,0 +1,41 @@
+﻿// // ------------------------------------------------------------------------------------------------
```

Comment:
I think we can do without this table. We could just use the Translation table... I'll fix that upon check-in.